pure-wasm/src/lib.rs

```diff
@@ -201,7 +202,10 @@ pub extern "C" fn check_enabled(engine_ptr: i32, message_ptr: i32, message_len:
 
         let toggle_name = ctx.toggle_name().expect("You need to pass a toggle name and you also need to remove this expect before production!");
 
-        let context = YggContext {
+        let context = EnrichedContext {
```

Comment:
Make use of the work in the last PR - we can now build a context directly and populate it with runtime_hostname